alacritty/build.rs

```diff
@@ -22,6 +22,10 @@ use std::path::Path;
 use embed_resource;
 
 fn main() {
+    if cfg!(not(any(feature = "x11", feature = "wayland", target_os = "macos", windows))) {
+        panic!("Please select at least one Linux/BSD backend using the corresponding features");
```

Comment:
```suggestion
        panic!("at least one of the "x11"/"wayland" features must be enabled");
```

I think we can be a bit shorter here and don't need to include Linux/BSD, since this error isn't emitted on Wayland/macOS anyways. I also think that we should definitely print the features we're talking about.

Reply:
Should we capitalize `At` at least?

Reply:
Please :)

Reply:
We shouldn't. Rust's panic messages are all lowercase without punctuation by default afaik, so we should follow that.

Reply:
@chrisduerr yep. https://github.com/rust-lang/api-guidelines/issues/79#issuecomment-316275971

Reply:
@nixpulvis I knew it's the case from using Rust for a bit, but I didn't know about the api-guidelines repository. That's an amazing resource to help with potential naming problems, thanks a ton for sharing.

Reply:
Sorry I moved the comment, below :P